### PR TITLE
add optional time_zone_name element to schedule XML schema

### PR DESCRIPTION
**Pull request by a-tze**
see #69

### CODE REVIEW

File: validator/xsd/schedule-without-person.xml.xsd [hunk 1]
```diff
@@ -15,6 +15,7 @@
               <xs:element type="xs:integer" name="days" minOccurs="0"/>
               <xs:element type="duration"   name="timeslot_duration" minOccurs="0"/>
               <xs:element type="httpURI"    name="base_url" minOccurs="0"/>
+	      <xs:element type="xs:string"  name="time_zone_name" minOccurs="0" maxOccurs="1"/>
```
Comment (johnjohndoe) [inline validator/xsd/schedule-without-person.xml.xsd:18]: Off-by one indentation if this is relevant for the XSD.

**Comment by johnjohndoe** [comment]
Is it possible to evaluate that the value of `time_zone_name` is one of a set of allowed values such as `Europe/Berlin`?

**Comment by a-tze** [comment]
> Is it possible to evaluate that the value of `time_zone_name` is one of a set of allowed values such as `Europe/Berlin`?

This was indeed my idea, after a quick research if there is something predefined in XSD (I couldn't find anything) it seems it must be defined manually as a choice, regex or whatever.

Another sub-aspect of this is, if it should only be a subset of this, e.g. IIRC there are 4 or more names for UTC. One possible thing might be to allow only non-deprecated labels.

File: validator/xsd/schedule-without-person.xml.xsd (hunk 1 above)
Comment (a-tze) [reply validator/xsd/schedule-without-person.xml.xsd]: thanks, fixed

**Review by saerdnaer — APPROVED** [review]
LGTM

**Comment by johnjohndoe** [comment]
Is there an instance of _frab_ or _Pretalx_ which has `time_zone_name` data which I use for implementing and testing?

**Comment by rixx** [comment]
This feature has not been contributed to pretalx as of yet.

**Pull request merged**